docs/src/pages/components/tag.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Tag
+group: Content
```

Comment:
Would move it to the forms, since it's a interactive component rather than read-only. Also has `label`, `description` and `error` 😀

Reply:
Ok done. In fact, I wasn't sure about the category.